docs/cody/clients/install-vscode.mdx

```diff
@@ -203,11 +210,19 @@ VS Code will typically notify you when updates are available for installed exten
 
 Cody also works with Cursor, Gitpod, IDX, and other similar VS Code forks. To access VS Code forks like Cursor, select **Sign in with URL and access token** and generate an access token. Next, copy and paste into the allocated field, using `https://sourcegraph.com` as the URL.
 
-## Using Ollama with Cody
+## Supported LLM models
+
+Cody offers a native support for Anthropic and Starcoder for Chat, Commands, and Autocomplete on the **Free** tier. Users on Cody **Pro** have the ability to choose from a list of supported LLM models for Chat, Commands, and Autocomplete. These LLMs are Claude Instant, Claude 2.0, Claude 2.1, Claude 3 (Opus and Sonnet), ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, and Mixtral.
+
+![llm-selection-cody-pro](https://storage.googleapis.com/sourcegraph-assets/blog/cody-vscode-1.8.0/claude3-2.png)
+
+Cody Enterprise users get additional capabilities like BYOLLM (Bring Your Own LLM) key. Your site administrator determines the LLM, and cannot be changed within the editor.
+
+## Supported local Ollama models with Cody
 
 <Callout type="info">Support with Ollama is currently in the Experimental stage and is available for Cody Free and Pro plans.</Callout>
 
-### Cody Autocomplete with Ollama
+### Cody autocomplete with Ollama
```

Comment:
I saw some inconsistency here with the capitalization of "autocomplete" (and chat etc.). Do we capitalize it (in which case this edit is maybe a typo) or not (in which case some of the other added paragraphs might have typos).

Reply:
Nice catch